Triangulation_2/doc/Triangulation_2/CGAL/Constrained_triangulation_2.h

```diff
@@ -9,6 +9,13 @@ Intersection tag for constrained triangulations, when input constraints do not i
 */
 struct No_intersection_tag{};
 
+/*!
+\ingroup PkgTriangulation2TriangulationClasses
+
+Intersection tag for constrained triangulations, when input constraints should not intersect.
+Will throw a `Intersection_of_constraints_exception` if constraints do intersect.
+*/
+struct Throw_on_intersection_tag{};
```

Comment:
Actually @lrineau suggested that we can use the existing `No_intersection_tag` so this tag becomes useless.